lib/fpm/package/deb.rb

```diff
@@ -428,26 +432,27 @@ def output(output_path)
       raise "#{name}: tar is insufficient to support source_date_epoch."
     end
 
+    attributes[:deb_systemd] = []
     attributes.fetch(:deb_systemd_list, []).each do |systemd|
       name = File.basename(systemd, ".service")
       dest_systemd = staging_path("lib/systemd/system/#{name}.service")
       mkdir_p(File.dirname(dest_systemd))
       FileUtils.cp(systemd, dest_systemd)
       File.chmod(0644, dest_systemd)
 
-      # set the attribute with the systemd service name
-      attributes[:deb_systemd] = name
+      # add systemd service name to attribute
+      attributes[:deb_systemd] << name
     end
 
-    if script?(:before_upgrade) or script?(:after_upgrade) or attributes[:deb_systemd]
+    if script?(:before_upgrade) or script?(:after_upgrade) or not attributes[:deb_systemd].empty?
```

Comment:
Similar style comment (no change required) - double negatives can be hard to read, and `or attributes[...].any?` to me is clearer than `not attributes[...].empty?`